src/xdocs/contributing.xml

```diff
@@ -169,5 +169,20 @@
         feel free to send a reminder email about your tracker item.
       </p>
     </section>
+
+    <section name="Bounty">
+
+      <p>
+        Some (mostly important) issues are marked with the <b>bounty</b> label.
```

Comment:
Please add link to gihub filter by bounty label.
A lot of user still unfamiliar with GitHub features.

Reply:
done